db/migrations/0002_user_order_ticket_alter_movie_actors_and_more.py

```diff
@@ -0,0 +1,114 @@
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
+            ],
+            options={
+                'verbose_name': 'user',
+                'verbose_name_plural': 'users',
+                'abstract': False,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Order',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+            ],
+            options={
+                'ordering': ['-created_at'],
+            },
+        ),
+        migrations.CreateModel(
+            name='Ticket',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('row', models.IntegerField()),
+                ('seat', models.IntegerField()),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='movies', to='db.Actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movies', to='db.Genre'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.cinemahall'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='movie',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.movie'),
+        ),
+        migrations.AddIndex(
+            model_name='movie',
+            index=models.Index(fields=['title'], name='db_movie_title_5d0841_idx'),
+        ),
+        migrations.AddField(
+            model_name='ticket',
+            name='movie_session',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.moviesession'),
+        ),
+        migrations.AddField(
+            model_name='ticket',
+            name='order',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.order'),
```

Comment:
Consider adding a `related_name` to the `order` ForeignKey in the `Ticket` model. This will help in making reverse queries from `Order` to `Ticket` more intuitive.